lineairdb_transaction.hh

```diff
@@ -48,12 +52,13 @@ public:
 private:
   LineairDB::Transaction* tx;
   LineairDB::Database* db;
+  std::string db_table_key;
```

Comment:
`db_table_key` is not initialized in constructor.
When you invoke any tx operations without `choose_table`, tx is supposed to ``read` and `write` with only a key: just like in normal LineairDB.
However, it may be better to return `nullptr` for `read` and `bool` for `write` in `lineairdb_transaction` class since we would probably never expect to invoke tx operations without `choose_table`.

Reply:
I understand. Thanks for explanation.

> we would probably never expect to invoke tx operations without `choose_table`.

Right.